hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2189,13 +2189,21 @@
     }
     {
       name: chip_sw_entropy_src_cs_aes_halt
-      desc: '''Verify the aes halt handshake with CSRNG.
+      desc: '''Verify the `entropy_src`'s aes halt handshake with CSRNG.
 
-            - Check that the AES block halts when the edn signals that sufficient entropy isn't
-              available to reseed via assertions.
-            - TODO: If the above assertion isn't covered... modify the above test to consume
-              entropy more quickly, by writing more quickly to the AES and KMAC and possibly
-              triggering reseeds by touching a block repeatedly.
+            - Through one of the application interfaces of CSRNG, issue a reseed command. This can
+              be done via a hardware peripheral requesting entropy via EDN.
+            - Through the SW application interface, issue a generate command.
+            - While the CSRNG is performing the generate operation, issue another reseed command
+              via the first application interface. Verify via backdoor probes, the SHA3 in the
+              entropy_src module is triggered at the same time the AES engine in CSRNG is
+              performing an operation.
+            - The second reseed command should trigger the entropy_src to request CSRNG to halt the
```

Comment:
maybe: this doesn't quite parse. I'd rewrite it as I interpret it "The second reseed command should trigger the entropy source to request that the CSRNG halts the AES engine until entropy is available."

Reply:
Done.